### PR TITLE
Added note in contributing guidelines

### LINKED ISSUE #356: Add Note in Contributing Guidelines
A lot of link titles include `|` and that causes an error in the rendering of the README on the website (refer to #352 ). We should add a note that all `|` should be replaced with another separator character.

### PR DESCRIPTION
**Pull request by wasp0094**
fixes #356 

image
preview.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ You need to add an emoji before the file name that represents what the resource
 
 2. After adding the commit title, click on propose changes and follow the steps to create a pull request.
 
+NOTE: Do not include `|` as a seperator for link titles since that causes errors in the rendering of the README on the website. Contributors are advised to replace it with some other similar seperator.
```
Comment (shahednasser) [inline CONTRIBUTING.md:86]: Can you add it before line 69?

**Review by shahednasser — CHANGES_REQUESTED** [review]
Great work @wasp0094 I just have one comment regarding the text placement.

**Comment by wasp0094** [comment]
hi @shahednasser I have applied new changes, pls review

**Comment by shahednasser** [comment]
Great work thanks!

**Pull request merged**